backend-api/app/schemas/consumers.py

```diff
@@ -2,7 +2,8 @@
 
 
 class ConsumerBase(BaseModel):
-    ethereum_address: str
+    eth_address: str
+    email: str
```

Comment:
Users shouldn't be able to change their email addresses (at least not yet). The backend uses the email as their username.